src/Nelmio/Alice/Fixtures/Loader.php

```diff
@@ -256,8 +261,8 @@ protected function parseFile($filename)
     /**
      * builds a collection of fixtures
      *
-     * @param  array      $rawData
-     * @return array
+     * @param  array     $rawData
```

Comment:
@theofidry $rawData is an array of class specs where key is class name and values is an array of specs where keys are some "names" and values are array of object property specs. If you can come up with good summary for all that it will be great.